.github/workflows/test-load-k6.yaml

```diff
@@ -0,0 +1,64 @@
+name: k6 Load Test
+
+on:
+  push:
+    branches:
+      - "main"
+      - "rc"
+      - "hotfix-rc"
+
+jobs:
+  run-tests:
+    name: Run k6 tests
+    runs-on: ubuntu-24.04
+    services:
+      # Datadog agent for collecting OTEL metrics from k6
+      datadog:
+        image: datadog/agent:7-full
+        ports:
+          - 4318:4318
+          - 5555:5555
+        env:
+          DD_SITE: us3.datadoghq.com
```

Comment:
❓ Use configuration vs. a fixed value?

Reply:
This leaves the door open for alternative Datadog sites, specifically EU.

Reply:
Sure, but do we want to make it a variable or secret?

Reply:
They aren't secrets, so a variable should work just fine.